### ISSUE TITLE
could you please give some advice about 'av' when I install pyslowfast

### ISSUE THREAD

**Issue by Tylerjoe** [facebookresearch/SlowFast#113]
I have followed the instruction of the INSTALL.md, and have installed all required library.
But when I built pyslowfast, I met this error as follow:

> (pytorch1.4) [root@45c384cb-d191-4529-94a3-4c5517344e0b slowfast]# python setup.py build develop
running build
running build_py
running develop
running egg_info
writing slowfast.egg-info/PKG-INFO
writing dependency_links to slowfast.egg-info/dependency_links.txt
writing requirements to slowfast.egg-info/requires.txt
writing top-level names to slowfast.egg-info/top_level.txt
reading manifest file 'slowfast.egg-info/SOURCES.txt'
writing manifest file 'slowfast.egg-info/SOURCES.txt'
running build_ext
Creating /usr/local/lib/python3.6/site-packages/slowfast.egg-link (link to .)
Adding slowfast 1.0 to easy-install.pth file
Installed /data/download/projects/video_recognition/slowfast
Processing dependencies for slowfast==1.0
Searching for av
Reading http://pypi.qiyi.virtual/simple/av/
Couldn't find index page for 'av' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading http://pypi.qiyi.virtual/simple/
No local packages or working download links found for av
error: Could not find suitable distribution for Requirement.parse('av')

But I have installed 'av' by `conda install av -c conda-forge`. I have not find a way to deal with this problem, could you please give some advices?

**Comment by takatosp1** [comment]
Hey @Tylerjoe welcome and thanks playing with PySlowFast!
Could you try to install av by PyPI? (pip install)

**Comment by Tylerjoe** [comment]
@takatosp1 Thanks for your kind and quick reply.
I have tried the method of your suggestion, but it has not worked for me. I don't know why I can't install 'av' by using `pip install av`, as there is a error when I use `pip install av`.
I have analyzed the problem. Finally, I have know the reason. Because I have create a virtual environment by `conda`, and pip is not linked with the python in the environment. After change it, it have worked!
This may help others if he met the same problem.
